include/fuzzuf/optimizer/templates/pso.hpp

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
Is this file still needed?

Reply:
fixed in 83ec6c5338d5a3177a259af6a6a0250b53214163